packages/fuel-gauge/src/e2e-script.test.ts

```diff
@@ -79,13 +79,13 @@ type MainArgs = [
  */
 describe('Live Script Test', () => {
   it('can use script against live Fuel Node', async () => {
-    if (!process.env.FUEL_NETWORK_GENESIS_KEY) {
+    if (!process.env.TEST_WALLET_PVT_KEY) {
```

Comment:
`TEST_WALLET_PVT_KEY` doesn't exist - did you mean to changed these?

Reply:
I forget to update the env name on the test workflow:

https://github.com/FuelLabs/fuels-ts/pull/1858/commits/019e378a44e72944a52691adbf52ab91c9a23844

`TEST_WALLET_PVT_KEY` will exist within the [test](https://github.com/FuelLabs/fuels-ts/blob/master/.github/workflows/test.yaml) workflow